packages/aws-cdk-lib/aws-stepfunctions/README.md

```diff
@@ -281,10 +284,12 @@ The `Pass` state also supports passing key-value pairs as input. Values can
 be static, or selected from the input with a path.
 
 The following example filters the `greeting` field from the state input
-and also injects a field called `otherData`.
+and also injects a field called `otherData`. You can also add custom state name
+to Pass state.
 
 ```ts
 const pass = new sfn.Pass(this, 'Filter input and inject data', {
+  stateName: 'my-pass-state',
```

Comment:
Leave this out I think? Or give a single example on `stateName`, adding a comment saying what the default is.